vmm/task/src/container.rs

```diff
@@ -195,6 +197,26 @@ impl KuasarFactory {
         Self { sandbox }
     }
 
+    // create_sandbox will do some preparation to provide a livable environment for containers,
+    // such as adding guest hooks, setting kernel paras, preparing sandbox files and namespaces
+    pub fn create_sandbox(&self) -> Result<()> {
```

Comment:
The things it do has nothing done with `create_sandbox`, I don't think it is a good idea to define such a function here. you can add a static mount and do it in main function, during system initialization

Reply:
I've split it to `early_init_call()` and `late_init_call()` in the second commit.